src/components/shared/StackSettings.js

```diff
@@ -146,32 +142,48 @@ class StackSettings extends PureComponent<Props> {
           </li>
           {hasAllocations && !disableCallTreeSummaryButtons ? (
             <li className="stackSettingsListItem stackSettingsFilter">
-              {this._renderCallTreeStrategyRadioButton(
-                'Timing',
-                'timing',
-                'Summarize using sampled stacks of executed code over time'
-              )}
-              {hasJsAllocations
-                ? this._renderCallTreeStrategyRadioButton(
-                    'JavaScript Allocations',
-                    'js-allocations',
-                    'Summarize using bytes of JavaScript allocated (no de-allocations)'
-                  )
-                : null}
-              {hasNativeAllocations
-                ? [
-                    this._renderCallTreeStrategyRadioButton(
-                      'Allocations',
-                      'native-allocations',
-                      'Summarize using bytes of memory allocated'
-                    ),
-                    this._renderCallTreeStrategyRadioButton(
-                      'Deallocations',
-                      'native-deallocations',
-                      'Summarize using bytes of memory deallocated'
-                    ),
-                  ]
-                : null}
+              <label>
+                Summarize:{' '}
```

Comment:
`Summarize:` looks okay to me, but pinging @julienw just in case he has ideas about it.

Reply:
I'm merging, but I can follow-up if we want to tweak this.